support/README.md

```diff
@@ -55,19 +55,19 @@ affected files manually or run the provided formatting script.
 
 To run the format fix script directly:
 
-```
-./tools/code_format/check_format.py fix && ./tools/code_format/format_python_tools.sh fix
+```shell
+bazel run //tools/code_format:check_format -- fix && ./tools/code_format/format_python_tools.sh fix
 ```
 
 To run the format fix script under Docker:
 
-```
+```shell
```

Comment:
we generally use `console` for these but i believe they are equivalent

Reply:
Done!